float32 deltaT
uint32 frameNo
uint32[2] spare